tests/scripts/tg/run_tg_unit_tests.sh

```diff
@@ -66,6 +66,24 @@ run_tg_llama3.1-70b_tests() {
   fi
 }
 
+run_tg_distributed_op_tests() {
+  # Record the start time
+  fail=0
+  start_time=$(date +%s)
+
+  echo "LOG_METAL: Running run_tg_distributed_op_tests"
+
+  pytest -n auto tests/ttnn/distributed/test_distributed_layernorm_TG.py ; fail+=$?
```

Comment:
I would recommend just 

```suggestion
  pytest tests/ttnn/distributed/test_distributed_layernorm_TG.py ; fail+=$?
```

because it's just one then we can get more logs